### PR TITLE
fix: /carbon-design-system#4217 keep page same

### PR DESCRIPTION
**Pull request by stvcisco**

### CODE REVIEW

**Comment by asudoh** [comment]
Hi 👋 thank you for creating a PR - This repository is for `v6` version of `carbon-componnts-react`. That said, do you want to send the same PR to https://github.com/carbon-design-system/carbon, too?

**Comment by stvcisco** [comment]
sorry about that - it's confusing with so many versions around.
I have found Pagination.js and Pagination-test.js under /carbon, but I am unable to do an npm install.  It fails because it cannot find a dependency in package.json: `eslint-config-carbon@10.4.0`
I do not see that version in npmjs.org - it's 2 years old at version 1.0.2.
I cannot run the jest tests until I can complete `npm install` of the project.

**Comment by asudoh** [comment]
I see, would you try `yarn install` instead of `npm install`?

**Comment by stvcisco** [comment]
PR created in /carbon: https://github.com/carbon-design-system/carbon/pull/4239

File: src/components/Pagination/Pagination.js [hunk 1]
```diff
@@ -155,17 +155,16 @@ export default class Pagination extends Component {
...
     return !pageSizesChanged && !pageChanged && !pageSizeChanged
       ? null
       : {
-          page: pageSizesChanged ? 1 : pageChanged ? page : currentPage,
-          pageSize: pageSizesChanged
-            ? pageSizes[0]
-            : pageSizeChanged
-            ? pageSize
-            : currentPageSize,
+          page: pageSizeChanged ? 1 : pageChanged ? page : currentPage,
```
Comment (emyarod) [inline src/components/Pagination/Pagination.js]:
```suggestion
          page: pageSizeChanged && 1 || pageChanged && page || currentPage,
```
pretty much the same comment as in https://github.com/carbon-design-system/carbon/pull/4239#discussion_r332034050

**Review by emyarod — COMMENTED** [review]
just to clarify do you mean to target `master` branch or `v6` branch?

File: src/components/Pagination/Pagination.js (hunk 1 above)
Comment (stvcisco) [reply src/components/Pagination/Pagination.js]: I intended to target master

**Comment by asudoh** [comment]
OK given this PR is for `master` let's close this for the sake of #4239.

**Pull request closed without merging**